src/pages/ErrorPage/ErrorPage.tsx

```diff
@@ -0,0 +1,14 @@
+import { useRouteError } from 'react-router-dom';
+
+export default function ErrorPage() {
+  const error = useRouteError();
+  console.error(error);
+
+  return (
+    <div id="error-page">
+      <h1>Oops!</h1>
+      <p>Sorry, an unexpected error has occurred.</p>
+      <p>{/* <i>{error.statusText || error.message}</i> */}</p>
```

Comment:
remove

Reply:
fixed: removed from code
TODO: find a way to define error type and implement <p<i>{error.statusText || error.message}</i></p>